README.md

```diff
@@ -108,27 +108,32 @@ The script accepts the following arguments:
 ```
 usage: leetcode_export [-h] [--cookies COOKIES] [--folder FOLDER]
                        [--problem-folder-name PROBLEM_FOLDER_NAME]
+                       [--no-problem-statement-files]
                        [--problem-statement-filename PROBLEM_STATEMENT_FILENAME]
                        [--problem-statement-content PROBLEM_STATEMENT_CONTENT]
                        [--submission-filename SUBMISSION_FILENAME]
-                       [--only-accepted] [--language LANGUAGE_UNPROCESSED] [-v]
-                       [-vv] [-V]
+                       [--only-accepted] [--only-latest-submission]
+                       [--language LANGUAGE_UNPROCESSED] [-v] [-vv] [-V]
 
 Export LeetCode submissions
 
-options:
+optional arguments:
```

Comment:
This is the help message that is displayed when the script is run with the `-h` flag (ex. `leetcode-export -h`). When I run it, the help message says `options` for me, is it the same for you?
If so, I would keep this text the same as the script help message

Reply:
> This is the help message that is displayed when the script is run with the `-h` flag (ex. `leetcode-export -h`). When I run it, the help message says `options` for me, is it the same for you? If so, I would keep this text the same as the script help message

In Python 3.9, it shows `optional arguments`.

Reply:
Thank you for pointing that out!

It looks like they've changed it to `optional` in Python 3.10. I would keep `options` because it's what will be used from now on.

**For reference:**
Python 3.9: https://github.com/python/cpython/blob/3.9/Lib/argparse.py#L1729
Python 3.10: https://github.com/python/cpython/blob/3.10/Lib/argparse.py#L1730